src/map/script.c

```diff
@@ -25454,6 +25467,36 @@ static void script_run_item_unequip_script(struct map_session_data *sd, struct i
 	script->current_item_id = 0;
 }
 
+static void script_run_item_rental_start_script(struct map_session_data *sd, struct item_data *data, int oid) __attribute__((nonnull(1, 2)));
```

Comment:
Why is there declaration of function here?

Reply:
it has a nonull attribute